### PR TITLE
FF104 Relnote: canvas fontkerning

### PR DESCRIPTION
**Pull request by hamishwillee**
FF104 added `CanvasRenderingContext2D.fontKerning` and`OffscreenCanvasRenderingContext2D.fontKerning` in https://bugzilla.mozilla.org/show_bug.cgi?id=1778908

The docs for `CanvasRenderingContext2D.fontKerning` were added in #19859 - waiting on the offscreen ones.

Other docs work for this can be tracked in #18772

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks Hamish, looks good. I guess this one depends on https://github.com/mdn/content/pull/19859 being merged first? There are some broken links 

```
    Can't resolve /en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack
    Can't resolve /en-US/docs/Web/API/CanvasRenderingContext2D/fontKerning
    Can't resolve /en-US/docs/Web/API/OffscreenCanvasRenderingContext2D/fontKerning
```

**Comment by hamishwillee** [comment]
@bsmth Yes :-).

You're having `OffscreenCanvasRenderingContext2D.fontKerning` at the same URL as the canvas version right? Should I update the URL to point to the same location?

Note that I don't normally wait on the required dependencies to merge release notes. But up to you.

File: files/en-us/mozilla/firefox/releases/104/index.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ No notable changes.
   Note that browsers may automatically provide visual indication on focused elements if the implementation determines that it will improve accessibility.
   (See {{bug(1765083)}} for more details.)
 
+#### Canvas
+
+- The `fontKerning` property is now supported on [`CanvasRenderingContext2D`](/en-US/docs/Web/API/CanvasRenderingContext2D/fontKerning) and [`OffscreenCanvasRenderingContext2D`](/en-US/docs/Web/API/OffscreenCanvasRenderingContext2D/fontKerning), enabling developers to specify how kerning is used when text is drawn to a canvas or offscreen canvas ({{bug(1778908)}}).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/104/index.md:50]: We could remove the `/fontkerning` URL path for OffscreenCanvas so we land on docs for that instead:

`.../API/OffscreenCanvasRenderingContext2D/fontKerning` -> `.../API/OffscreenCanvasRenderingContext2D`

what do you think?

```suggestion
- The `fontKerning` property is now supported on [`CanvasRenderingContext2D`](/en-US/docs/Web/API/CanvasRenderingContext2D/fontKerning) and [`OffscreenCanvasRenderingContext2D`](/en-US/docs/Web/API/OffscreenCanvasRenderingContext2D), enabling developers to specify how kerning is used when text is drawn to a canvas or an offscreen canvas ({{bug(1778908)}}).
```

**Comment by bsmth** [comment]
Hi @hamishwillee 

> You're having `OffscreenCanvasRenderingContext2D.fontKerning` at the same URL as the canvas version right? 

Yes, the offscreen variant references `CanvasRenderingContext2D.fontKerning`. I've left a suggestion to link to the offscreen context docs, let me know what you think 👍🏻

**Review by bsmth — APPROVED** [review]
Thanks 👍🏻

**Comment by bsmth** [comment]
I'm just going to merge, looks good from my side

**Comment by hamishwillee** [comment]
Thanks @bsmth Looks good to me! Note I'm around Mon, Tue, Fri AEST. So there is often a 3 day wait on responses. If you think something is sensible then just merging as you have done is fine :-)

**Pull request merged**